mlir/lib/Dialect/Rock/Transforms/OutputSwizzle.cpp

```diff
@@ -441,14 +426,6 @@ void RockOutputSwizzlePass::runOnOperation() {
                    << ldsRequiredBytes << " bytes, skipping pass\n");
         return;
       }
-      // heuristic: if we need more LDS, skip this pass
```

Comment:
we should check if there's any performance regression due to this. I'm happy to do this if you are busy with other things.